content/en/docs/concepts/containers/images.md

```diff
@@ -63,6 +63,21 @@ Some image name examples that Kubernetes can use are:
 - `registry.k8s.io/pause@sha256:1ff6c18fbef2045af6b9c16bf034cc421a29027b800e4f9b68ae9b1cb3e9ae07` - Image name with digest.
 - `registry.k8s.io/pause:3.5@sha256:1ff6c18fbef2045af6b9c16bf034cc421a29027b800e4f9b68ae9b1cb3e9ae07` - Image name with tag and digest. Only digest will be used for pulling.
 
+## Image content
+
+OCI images are composed of layers. Each layer is a set of files that are added to the image.
```

Comment:
```suggestion
Container images are composed of _layers_. Each layer is a set of files that are added to the image.
```

Docker-format images have layers too. It'd be wrong to imply that only OCI images have layers.